walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,97 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    // Base cases that should return default results
+    // if there is no attendee, return the whole day as available
+    Collection<String> attendees = request.getAttendees();
+    Collection<String> optionalGuests = request.getOptionalAttendees();
```

Comment:
I think it may be because they get MeetingRequest from the client end. In the client end, the attendees are stored as an ArrayList, so they had to use as broad of a definition as possible.